### ISSUE TITLE
Ordre des contenus touristiques

### ISSUE THREAD

**Issue by pnrma** [GeotrekCE/Geotrek-mobile#305]
Sur les fiches rando, l'affichage des contenus touristiques ne semble pas prendre en compte l'ordre des catégories spécifié côté Geotrek-admin.

**Comment by bastyen** [comment]
Effectivement, la propriété order des catégories des contenues touristiques n'est pas pris en compte.

Je viens d'apprendre son existence et je ne sais pas si elle était présente lors du développement de cette partie de l'application.

**Comment by pnrma** [comment]
Serait-il possible d'inclure ce tri lors d'une prochaine mise à jour ou cela demande un financement spécifique ?

**Comment by babastienne** [comment]
Bonjour @pnrma, pour réaliser cette évolution il faudrait en effet prévoir un financement car cela implique de développer exprès un morceau de code dédié à ce fonctionnement, à la fois dans l'application mobile et dans Geotrek-Admin. 
N'hésitez pas à revenir vers nous si vous êtes intéressé.
